community/cypher/docs/cypher-docs/src/docs/dev/ql/using/index.asciidoc

```diff
@@ -18,9 +19,12 @@ It will force Cypher to not use an index that could have been used, and instead
 
 NOTE: You cannot use index hints if your query has a `START` clause.
 
+You can also make Neo4j produce plans that perform joins between query sub-graphs.
```

Comment:
If we are referring to join hints, I think we should be consistent with the above word 'force', like so: `You can also force Neo4j to produce plans which perform joins between query sub-graphs.`